0.6.1/docker-entrypoint.sh

```diff
@@ -61,4 +61,9 @@ elif vault --help "$1" 2>&1 | grep -q "vault $1"; then
     set -- vault "$@"
 fi
 
+# If we are running Vault, make sure it executes as the proper user.
+if [ "$1" = 'vault' ]; then
```

Comment:
Indeed! Normalizing it like in Consul would be safer so not to break anything after user is switched, on current installations.